frameworks/cassandra/docs/release-notes.md

```diff
@@ -7,6 +7,20 @@ menuWeight: 120
 
 ---
 
+## Version 2.1.2-3.0.15-beta
+
+## NOTICE
+
+This is a beta release of the DC/OS Apache Cassandra framework. It contains multiple improvements as well as new features that are to be considered of beta quality. Do _not_ operate this version in production.
+
+### Improvements
+- Remove the dependency on a docker image and allow the service to run as nobody in CentOS distros. [#2067](https://github.com/mesosphere/dcos-commons/pull/2067)
+- Moved scheduler health-checks from HTTP to [Mesos-level health checks](https://mesosphere.github.io/marathon/docs/health-checks.html#mesos-level-health-checks).
```

Comment:
How does this impact an operator?

Reply:
@gabrielhartmann might have more details, but from what I understand the biggest impact of switching to MESOS_HTTP is decreasing the number of false positives due to networking failures during health checks, and also moving health-checking overhead from the scheduler to the agents running service tasks.